types/d3-delaunay/d3-delaunay-tests.ts

```diff
@@ -11,24 +11,49 @@ import * as d3 from 'd3-delaunay';
 // Test Delaunay class
 
 const constructedDelaunay = new d3.Delaunay([0, 1, 2, 3, 4, 5, 6, 7, 8, 9]);
-const defaultDelaunayFromArray: d3.Delaunay<[number, number]> = d3.Delaunay.from([[0, 0], [1, 0], [0, 1], [1, 1]]);
-const defaultDelaunayFromIterator = d3.Delaunay.from((function*() {
-    yield [0, 0];
-    yield [1, 0];
-    yield [0, 1];
-    yield [1, 1];
-})());
-const customDelaunayFromArray = d3.Delaunay.from({length: 4}, (d, i) => i & 1, (d, i) => (i >> 1) & 1);
-const customDelaunayFromIterator = d3.Delaunay.from((function*() {
-    yield {x: 0, y: 0};
-    yield {x: 1, y: 0};
-    yield {x: 0, y: 1};
-    yield {x: 1, y: 1};
-})(), p => p.x, p => p.y);
+const defaultDelaunayFromArray: d3.Delaunay<[number, number]> = d3.Delaunay.from([
+    [0, 0],
+    [1, 0],
+    [0, 1],
+    [1, 1],
+]);
+const defaultDelaunayFromIterator = d3.Delaunay.from(
+    (function*() {
+        yield [0, 0];
+        yield [1, 0];
+        yield [0, 1];
+        yield [1, 1];
+    })(),
+);
+const customDelaunayFromArray = d3.Delaunay.from(
+    { length: 4 },
+    (d, i) => i & 1,
+    (d, i) => (i >> 1) & 1,
+);
+const customDelaunayFromIterator = d3.Delaunay.from(
+    (function*() {
+        yield { x: 0, y: 0 };
+        yield { x: 1, y: 0 };
+        yield { x: 0, y: 1 };
+        yield { x: 1, y: 1 };
+    })(),
+    p => p.x,
+    p => p.y,
+);
+const validContext = {
+    moveTo: (x: number, y: number) => {}, // Delaunay.MoveContext
+    lineTo: (x: number, y: number) => {}, // Delaunay.LineContext
+    closePath: () => {}, // Delaunay.ClosableContext
+    arc: (x: number, y: number, radius: number, startAngle: number, endAngle: number, counterClockwise?: boolean) => {}, // Delaunay.ArcContext
+    rect: (x: number, y: number, width: number, height: number) => {}, // Delaunay.RectContext
+};
+const invalidContext = {
+    moveTo: (x: number, y: number) => {},
+};
```

Comment:
Added these context objects to test several overloaded methods that were previously only testing the no argument version.